src/data/colors.ts

```diff
@@ -13,5 +13,5 @@ export default {
   "yellow-regular": "#fdefd1",
   "yellow-medium": "#fce1a8",
   "yellow-bold": "#f1af1b",
-  "green-regular": "#0eda83",
+  "green-regular": "#0eda83"
```

Comment:
I think we do want the trailing comma here, because the convenience of it is that you can just start typing a new line without having to think about adding a comma to the previous line, right?